pkg/osd_cluster_upgrader/upgrader_test.go

```diff
@@ -205,8 +238,18 @@ var _ = Describe("ClusterUpgrader", func() {
 					Items: []configv1.ClusterVersion{
 						{
 							Spec: configv1.ClusterVersionSpec{
+								Channel: upgradeConfig.Spec.Desired.Channel,
 								DesiredUpdate: nil,
 							},
+							Status: configv1.ClusterVersionStatus{
+								AvailableUpdates: []configv1.Update{
+									{
+										Version: upgradeConfig.Spec.Desired.Version,
+										Image:   "quay.io/this-doesnt-exist",
```

Comment:
Does the "this-doesnt-exist" have meaning for the test?

Reply:
It doesn't have meaning - I've renamed it to make it look less like it's trying to test some edge case around image availability.